### PR TITLE
PR for #27: Implement github actions

### PR DESCRIPTION
**Pull request by santiagohermo**
Can you take a quick look here @jmshapir? I implemented a few changes to the repo to bring back GitHub Actions.

Tasks:
- Confirm that changes in code look good (they were created by the `usethis` package)
- Decide whether we want to get rid of the old actions/workflow
  - As explained [here](https://github.com/JMSLab/eventstudyr/issues/27#issuecomment-1483042849), actions v2 changed the name of the `yaml` file. Thus, we now have [`actions/workflows/check-standard.yaml`](https://github.com/JMSLab/eventstudyr/actions/workflows/check-standard.yaml) (new) and [`actions/workflows/R-CMD-check.yaml`](https://github.com/JMSLab/eventstudyr/actions/workflows/R-CMD-check.yaml) (old).
  - I think that disabling the old workflow should be fine, as it mostly contains runs that consisted of a "simple task" unrelated to the project. To disable:

  image

Let me know if you prefer @nateschor to take over this PR!

### CODE REVIEW

**Comment by nateschor** [comment]
@santiagohermo before we merge this into main, do we need to change the [release](https://github.com/JMSLab/eventstudyr/releases) to `v1.0.1` as you described [here](https://github.com/JMSLab/eventstudyr/pull/29#issuecomment-1479702022)? If so, would you be able to implement?

File: .Rbuildignore
Comment (nateschor) [inline .Rbuildignore:9]: I'm not sure that we want to have the entire `vignettes` folder included? If you look at `ggplot2`, they have [some directories](https://github.com/tidyverse/ggplot2/blob/375a24b58c156c2bbb5bf18a027461e2a03d4560/.Rbuildignore#L25-L26) listed in the `.Rbuildignore`, but not [all](https://github.com/tidyverse/ggplot2/tree/375a24b58c156c2bbb5bf18a027461e2a03d4560/vignettes) of them 

```suggestion

```

**Comment by jmshapir** [comment]
> * I think that disabling the old workflow should be fine, as it mostly contains runs that consisted of a "simple task" unrelated to the project. To disable:

@santiagohermo if the old workflow is superseded I'm on board with removing, and will review the rest shortly thanks!

**Review by jmshapir — APPROVED** [review]

File: .Rbuildignore
Comment (santiagohermo) [reply .Rbuildignore]: Not sure what's the best here. In any case, I implemented your suggestion in https://github.com/JMSLab/eventstudyr/pull/30/commits/e43ce94b1cb14f8d2ce45cf2187864cf6cb28475 and Actions ran correctly after the commit. Workflow run is [here](https://github.com/JMSLab/eventstudyr/actions/runs/4515115218)

**Comment by santiagohermo** [comment]
> @santiagohermo before we merge this into main, do we need to change the [release](https://github.com/JMSLab/eventstudyr/releases) to `v1.0.1` as you described [here](https://github.com/JMSLab/eventstudyr/pull/29#issuecomment-1479702022)? If so, would you be able to implement?

The goal was not to change the release @nateschor, but rather to create a new one with the fixes. Since this issue is almost done I think we can incorporate it as well.

A draft for the release is [here](https://github.com/JMSLab/eventstudyr/releases/tag/untagged-c9219de8e4f5e0391a89). Once published, the release will create the `v1.0.1` tag pointing to `main`. Thus, we must publish after merging this issue! Other things to incoporate to the release:

- [ ] Update the zip file created with `devtools::build()`.
- [ ] Change XXXX with the latest commit to `main` after finishing this issue.

Do I have your green light to follow this plan @nateschor?

**Comment by nateschor** [comment]
> The goal was not to change the release @nateschor, but rather to create a new one with the fixes

@santiagohermo sorry, yes, create a new one--thanks for catching that!

 > Thus, we must publish after merging this issue!

Thanks for clarifying!

> Do I have your green light to follow this plan @nateschor?

My only question is if it's common practice with the release notes to give general information about the changes ("implements minor fixes") rather than to give more details about what was changed (removed git lfs for `.rda` files in example data, refactored code that created example data)?

All of these additional details are accessible in the closing issue summary comments and I see you liked the PRs for the release, so I'm not sure if it's necessary?

As long as we don't need to provide more information about the release changes, you have the green light from me! Thanks @santiagohermo!

**Comment by santiagohermo** [comment]
Thanks @nateschor! I changed the draft to look more like ggplot2 releases ([example](https://github.com/tidyverse/ggplot2/releases/tag/v3.4.1)). The new version is [here](https://github.com/JMSLab/eventstudyr/releases/tag/untagged-faf29715e4c38db2485d). Does this look better?

**Comment by santiagohermo** [comment]
Thanks @nateschor! 

- Release is [here](https://github.com/JMSLab/eventstudyr/releases/tag/v1.0.1)
- Summary of #27 is [here](https://github.com/JMSLab/eventstudyr/issues/27#issuecomment-1483520591)

**Pull request merged**
